2023-11 Changelog Entry

trevorgerhardt opened the pull request:
This references some changes made in https://github.com/conveyal/docs/pull/79, so that PR should be merged first.

trevorgerhardt committed:
Initial commit of new changelog entry

ansoncfit committed:
Add November 2023 changelog

ansoncfit committed:
Add November 2023 changelog

abyrd commented on changelog/2023-11-22.mdx:
```diff
@@ -0,0 +1,22 @@
+  alt='Smoother regional analysis view'
+/>
+
+We have been hard at work on Conveyal's major release for 2023. This release includes **v7.0** of the R5 routing engine and...
```
Will this wording draw more attention to the fact that there are not many releases in 2023? Maybe we should say something like "Conveyal's late 2023 major release."

trevorgerhardt replied:
I agree, I suggest "November, 2023" as that would fit the style that we've used in the most recent changelog entries.

abyrd commented on changelog/2023-11-22.mdx:
```diff
@@ -0,0 +1,22 @@
+---
+title: 2023 Major Release
```
See comment below.

abyrd reviewed (commented):
Wording of the changelog entries themselves looks good. Seeing the title in situ, I'm thinking we should de-emphasize the singular nature of the release. We could stick a more specific modifier in front of 2023 which would still be accurate, but not cause people to read and judge the feature list as harshly.

ansoncfit committed:
Update changelog

- De-emphasize major release by using month/date naming convention
- Revise vector map and dual access change summaries

ansoncfit committed:
Add reference to TransitCenter equity dashboard
